.github/workflows/update_website.yml

```diff
@@ -29,70 +25,33 @@ jobs:
           git clone git@github.com:kerzendorf-group/groupwebsite_generator.git
           git clone git@github.com:kerzendorf-group/kerzendorf-group.github.io.git
```

Comment:
can you use this action-
https://github.com/actions/checkout

Reply:
done